omnibus/config/software/datadog-agent.rb

```diff
@@ -178,6 +178,20 @@
           mode: 0644,
           vars: { install_dir: install_dir, etc_dir: etc_dir }
     end
+    if suse?
+      erb source: "sysvinit_suse.erb",
+          dest: "#{install_dir}/scripts/datadog-agent",
+          mode: 0755,
+          vars: { install_dir: install_dir, etc_dir: etc_dir }
```

Comment:
Is there any reason to use ERB templates? I don't see us using `install_dir` nor `etc_dir` in them. If it's just because this is a copy-paste it's fine, just making sure it wasn't an oversight and you actually wanted to use these variables.

Reply:
I copy-pasted these, and yeah I'm not using them in the template. Moving forward, I think we should use them (in all template files, not only these ones), as we are currently hardcoding the installation directory and the etc directory in the init files, such as here: https://github.com/DataDog/datadog-agent/pull/4436/files#diff-6f0265d9199c0ebe5576a5d949cccb40R13-R14